### DISCUSSION TITLE
Adding more than one keyword to blacklist meetings

### DISCUSSION THREAD

**Discussion by skyshines** [TobiasPankner/Teams-Auto-Joiner#166] [General]
in the config json
"blacklist_meeting_re" allows only to write only one keyword
can we add more than one?

like "blacklist_meeting_re" = "A-1"
more like add A-2 A-3 etc too
any way to do this?

**Comment by skyshines** [comment]
@TobiasPankner can you look into this pls

**Comment by TobiasPankner** [comment]
The use-case you described is exactly what regular expressions (what the "re" stands for) do.  
For example: `A-\d` would match "A-1", ...  "A-9". Similar to this simple example you can build any matcher you want. 

It can get quite difficult for larger matcher, here is a helpful page https://regexr.com/

**Reply by skyshines** [reply]
Well tbh A-1 is my batch
If it's get blacklisted
There will be problem
😅

I don't want to attend lectures from
A-2 A-3
Sometimes
A-|| or A-|||

Or 
A2 A3

**Reply by TobiasPankner** [reply]
Okay. Well the regex for that would be:   `(A-[23])|(A-\|{1,2})|(A[23])`  
Although I am not a regex expert.

**Reply by TobiasPankner** [reply]
In the middle it's actually {2,3}
